### ISSUE TITLE
Systemctl start timeout

### ISSUE THREAD

**Issue by IshanArya** [plexdrive/plexdrive#405]
Whenever i try to run `sudo systemctl start plexdrive.service` i get this error: `job for plexdrive.service failed because a timeout was exceeded`

in `systemctl status plexdrive.service` i get the notification that `First cache build process finished!` but the service times out a few minutes later either way

I tried changing up the startup time to about 6 minutes as well but it still fails. Any idea why?

Here is my plexdrive.service:
```
# /etc/systemd/system/plexdrive.service

[Unit]
Description=plexdrive
After=network-online.target
AssertPathIsDirectory=/mnt/plexdrive

[Service]
Type=notify
Restart=always
TimeoutStartSec=400
ExecStart=/usr/bin/plexdrive mount --drive-id=xxxxxxx -o allow_other -v 2 --chunk-load-threads=16 --chunk-load-ahead=16 --max-chunks=256 --refresh-interval=1m /mnt/plexdrive
ExecStopPost=/usr/bin/fusermount -quz /mnt/plexdrive

[Install]
WantedBy=multi-user.target
```

**Comment by IshanArya** [comment]
I realized the problem was that I set Type=notify and fixed it by setting Type=simple. However, the service file in the [tutorial](https://github.com/plexdrive/plexdrive/blob/master/TUTORIAL.md) uses Type=notify so i'm not sure why this doesn't work...

**Comment by IshanArya** [comment]
Is there a release schedule for when master branch will be turned into an official release?

**Comment by felixbuenemann** [comment]
Not really, I should probably do one soon, now that the new cache backend including disk cache is merged.

But if you have go 1.12 or newer installed, a git clone followed by "go build" should be enough.

**Comment by felixbuenemann** [comment]
I've just released plexdrive 5.2.0 with all of the accumulated changes.

**Comment by IshanArya** [comment]
awesome thanks!
